imagedephi/main.py

```diff
@@ -24,10 +25,18 @@ class ImagedephiContext:
     override_rule_set: RuleSet | None = None
 
 
+CONTEXT_SETTINGS = {"help_option_names": ["--help"]}
+if sys.platform == "win32":
+    # Allow Windows users to get help via "/?".
+    # To avoid ambiguity with actual paths, only support this on Windows.
```

Comment:
The other option would be to check if it is an actual path and use it; if not, give help.

Reply:
I want to avoid these sort of decision trees in application logic.

On Windows, this will indeed mean it's impossible to specify an input path of `/?`, but I think it's exceedingly unlike that such a path would ever exist.